src/zones/zonesWithPoints.ts

```diff
@@ -27,19 +27,19 @@ export interface FromToWithNumberOfPoints extends FromTo {
  * @param options - options
  * @returns array of zones with points
  */
-export function zonesWithPoints(
-  zones: FromTo[] = [],
+export function zonesWithPoints<Zone extends FromTo = FromTo>(
+  zones: Zone[] = [],
```

Comment:
Zone is not used in this method and we don't return extended Zone (with points). Does not make sense to me.